package.json

```diff
@@ -62,6 +62,7 @@
     "express": "^4.13.3",
     "extract-text-webpack-plugin": "^0.9.1",
     "file-loader": "^0.8.5",
+    "gzip-js": "0.3.2",
```

Comment:
`gzip-js` code hasn't been implemented in AlternateDownloader yet; if the zipping solution is ultimately unused, someone please prod me and remind me to remove this.